rohmu/delta/common.py

```diff
@@ -143,11 +143,18 @@ class SnapshotFile(DeltaModel):
     mtime_ns: int
     hexdigest: str = ""
     content_b64: Optional[str]
+    # Indicator if a file should be part of a chunk/bundle - useful for the files which are small to be treated as
+    # separate delta hash files, but not small enough to be embedded into a manifest file, so better to group them
+    # together when e.g. uploading to the object storage
+    is_bundled: bool = False
```

Comment:
I'd change this name to something like ```should_be_bundled``` rather than ```is_bundled``` or maybe just ```bundle```, because we don't actually do the processing in this class, so it's not a property of the object, right?